Update installation instructions

ordepdev opened the pull request:
Closes #54.

ordepdev commented:
Should we merge this @SeanTAllen?

SeanTAllen commented:
The default installation path for ponyup has changed and varies according to OS. At the moment, there is only Linux prebuilt versions for releases but nightlies are available for macOS and Linux.  Windows still requires building from source. 

I think those details need to be accounted for.

ordepdev commented:
Finally I was able to update this PR with your notes @SeanTAllen 🙏

SeanTAllen commented on README.md:
````diff
@@ -10,7 +10,25 @@ Production ready.
+### Ponyup
+
+You can use [`ponyup`](https://github.com/ponylang/ponyup#installing-ponyup), our toolchain multiplexer, to download both `release` and `nighly` builds, if you're using either Linux or macOS. Windows still requires building from source
+
+Linux only:
+
+```bash
+ponyup update changelog-tool release
+```
````
```suggestion

Linux only:

```bash
ponyup update changelog-tool release
```
```

SeanTAllen commented on README.md:
````diff
@@ -10,7 +10,25 @@ Production ready.
+
+You can use [`ponyup`](https://github.com/ponylang/ponyup#installing-ponyup), our toolchain multiplexer, to download both `release` and `nighly` builds, if you're using either Linux or macOS. Windows still requires building from source
+
+Linux only:
+
+```bash
+ponyup update changelog-tool release
+```
+
````
This should be deleted.

ordepdev replied:
Dammit, I've made a mistake, it should be:

```
Linux only:
ponyup update changelog-tool release
```

```
Linux and macOS:
ponyup update changelog-tool nightly
```

Searching for the release packages on [cloudsmith](https://cloudsmith.io/~ponylang/repos/releases/packages/?q=changelog-tool), the latest release exists for Linux Release and nightlies for Linux and only nightlies for macOS as described in the original issue, still applies? Otherwise, I'll leave only the latter installing instruction.

SeanTAllen replied:
Why do you have Linux twice?

SeanTAllen replied:
There's still only Linux. There's been no release of changelog-tool since that time.

ordepdev replied:
Ok, I was under the impression that we could only install `release` in Linux, and `nightly` on both Linux and macOS.

SeanTAllen replied:
Thats correct but its confusing to give people two options.

SeanTAllen commented:
@ordepdev this will also need to be rebased against master to pickup conflicting changes as README was updated since you started working on this last year.

Commit:
Update installation instructions

This patch points the installation to Linux release builds and Linux/macOS
nighly builds using our toolchain multiplexer, ponyup.

Closes #54.

ordepdev commented:
Rebased 🙏

SeanTAllen commented:
Thanks
